yarn-project/ethereum/src/config.ts

```diff
@@ -125,31 +125,32 @@ const StagingPublicGovernanceConfiguration = {
 
 const TestnetGovernanceConfiguration = {
```

Comment:
I think the values are mostly sane, but would have preferred seeing it not explode in tests first (when trying to update some of these in the #16988 I ran into plenty things)